packages/ui/src/lib/NativeBase/theme.ts

```diff
@@ -0,0 +1,21 @@
+import { extendTheme } from "native-base";
+
+export default extendTheme({
+  colors: {
+    white: "#FFFFFF",
+    black: "#222222",
```

Comment:
Is `#222222` from our colour palette?

Reply:
Yes, I found this color from https://github.com/UTOFUN/utofun/blob/develop/clients/less/_navbar.less#L61=.

Reply:
strange :(

Reply:
Cool, I didn't know we have a color palette on wiki. BTW, I have a question, where do [the colors in pokemon](https://github.com/UTOFUNdevelop/pokemon/blob/develop/packages/flareon/src/ui/Provider.tsx#L9-L18=) come from? They are not on the wiki.

Reply:
After comparing the color on the current website and the wiki, I found there are many differences between them. Perhaps we need to reselect the colors on the palette.

Reply:
I see. I will first use the colors in Figma at the moment.

Reply:
sorry to have confused you, figma is not our ssot yet, please use provider as our ssot for now